lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -198,6 +198,31 @@ def get_release_url(repository:, tag_name:)
         nil
       end
 
+      # Publishes an existing GitHub Release still in draft mode. Note that this will search only within the first page of releases, as returned by the GitHub API.
```

Comment:
🤔  Are we sure about this? The `initialize` of this helper sets `@client.auto_paginate = true` on line 25, so I think that would not stop at first page?

Reply:
Ah, that's true, nice catch!
That wasn't the case when I was using the `github_api` action directly from the app side and started with it here.
Indeed `auto_paginate` from Octokit [will concatenate the pages into a single array](https://github.com/octokit/octokit.rb?tab=readme-ov-file#auto-pagination).

Reply:
```suggestion
      # Publishes an existing GitHub Release still in draft mode.
```